model-optimizer/extensions/ops/elementwise.py

```diff
@@ -22,13 +23,49 @@ def override_data_type_of_constant(node: Node):
         # power value 2, or when replacing Neg operation with Mul with -1 as second input.
         in_node_0 = node.in_port(0).get_source().node
         in_node_1 = node.in_port(1).get_source().node
-        if in_node_0.op == 'Const':
-            convert_const_node_value_type(in_node_0, in_type_1)
-        elif in_node_1.op == 'Const':
-            convert_const_node_value_type(in_node_1, in_type_0)
+
+        if in_node_0.soft_get('op') != 'Const' and in_node_1.op != 'Const':
+            raise Exception("Elementwise operation '{}' has inputs of different data types: '{}' and '{}' "
+                            'that cannot be aligned'.format(node.soft_get('name'), in_type_0, in_type_1))
```

Comment:
use the same quotes . Now you use both double quotes and the single quotes in one string (line 28 and 29). Looks ugly.

Reply:
done